src/apis/data/endpoints.py

```diff
@@ -118,7 +125,10 @@ class CityCasesList(Resource):
     @data_endpoints.marshal_with(city_cases_response_list)
     def get(self):
         """Cases per city list"""
-        return city_services.get_city_cases(None)
+        response = city_services.get_city_cases(None)
+        if not response['cases']:
```

Comment:
here you will get a `KeyError` if the key did not exist. So you could use `get()` to return `None`
```suggestion
        if not response.get('cases'):
```